src/models.c

```diff
@@ -629,6 +629,27 @@ void DrawCylinderWires(Vector3 position, float radiusTop, float radiusBottom, fl
     rlPopMatrix();
 }
 
+//Draw an arc between two points
+//If iradius=0. the arc is a semi-circle with a center at the midpoint of the line between the two points.  Increase iradius for a flatter arc with a center off the midpoint.  Theta rotates the arc around the line between the two points.
+void DrawArc(Vector3 v1, Vector3 v2, float iradius, float theta, int steps, Color color) 
```

Comment:
I understand how this works but, sincerely, `iradius` and `theta` do not seem intuitive for the users, maybe is the parameter name...

Reply:
@raysan5 : That makes sense.  Instead of `theta` would simply `rotation` be better?  And for uniformity, it will be degrees not radians (and use `DEG2RAD` for conversion).

Your critique of `iradius` made me think about what might be most useful.  What about this?

Replace `iradius` with `flatness`.  Make `flatness` relative to the distance between the two points.  A flatness of 1.0 is the minimum radius = half the distance between the points = a semi-circle.  A flatness of 2.0 means twice the minimum radius, so an arc that's about twice as "flat" as a semi-circle.  This has the advantage of providing the same shape of the curve at any scale of distance between the two points.  The user can still find the arc radius easily:  Arc radius= (Half distance between the two points) * flatness

Reply:
@ProfJski I decided to investigate how other libraries implement this function, note that most of them refer to `DrawArc()` as a 2D arc, so, `DrawArc3D()` would be a possibility. Here some use cases:

 - [Handles.DrawWireArc](https://docs.unity3d.com/ScriptReference/Handles.DrawWireArc.html) Unity Engine
```cs
void DrawWireArc(Vector3 center, Vector3 normal, Vector3 from, float angle, float radius, float thickness = 0.0f);
```
 - [DrawArc](https://docs.unrealengine.com/4.26/en-US/API/Runtime/Engine/DrawArc/) Unreal Engine
```cpp
void DrawArc(FPrimitiveDrawInterface * PDI, const FVector Base, const FVector X, const FVector Y, const float MinAngle, const float MaxAngle, const float Radius, const int32 Sections, const FLinearColor & Color, uint8 DepthPriority);
```
 - [DrawCircleArc](https://docs.godotengine.org/en/stable/tutorials/2d/custom_drawing_in_2d.html#an-example-drawing-circular-arcs) Godot Engine example
```cs
void DrawCircleArc(Vector2 center, float radius, float angleFrom, float angleTo, Color color);
```
 - [Graphics.DrawArc Method](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.graphics.drawarc?view=net-5.0) .NET System.Drawing:
```cs
void DrawArc(System.Drawing.Pen pen, System.Drawing.Rectangle rect, float startAngle, float sweepAngle);
```
 - [Canvas.drawArc](https://developer.android.com/reference/android/graphics/Canvas#drawArc(android.graphics.RectF,%20float,%20float,%20boolean,%20android.graphics.Paint)) Android Canvas ([+info](https://thoughtbot.com/blog/android-canvas-drawarc-method-a-visual-guide))
```java
void drawArc(RectF oval, float startAngle, float sweepAngle, boolean useCenter, Paint paint);
```

What do you think?

Reply:
I keep thinking about it... maybe instead of `flatness` we can call it `roundness` so, a `roundness = 0.0` would be a flat line (and `theta` not considered) and a `roundness = 1.0` would be a semi-circle, even allowing greater values for bigger circles. About `theta`, maybe we can call it `swinging` so, with `swinging = 0.0` the arc is balanced and then allow `swinging = -1.0` and `swinging = 1.0` to balance the arc to the left (first vertex) or to the right (second vertex).

What do you think?

Reply:
I would say iRadius and theta are more intuitive than making up new names for things, especially when speaking about well established mathematical concepts like arcs in 2d and 3d space. 

How would "flatness" and "swinging", two terms which I have never heard in relationship to an arc, be more intuitive to and end user?

If you wanted to make it more intuitive to end users, it should probably follow a similar convention to pretty much everything else out there instead of trying to make up something new. Which means you have a radius, fromAngle, and toAngle.